src/expr/src/relation/mod.rs

```diff
@@ -1318,53 +1318,51 @@ impl MirRelationExprVisitor {
         F: FnMut(&'a MirRelationExpr) -> Result<(), E>,
         E: From<RecursionLimitError>,
```

Comment:
In theory this constraint could be removed/relaxed now, couldn't it?

Reply:
Good catch.